addons/withWeb/withWeb.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
```

Comment:
I would move all of the contents of this file to `index.js` and remove this file.